mda_openbabel_converter/OpenBabelParser.py

```diff
@@ -4,26 +4,119 @@
 
 import MDAnalysis as mda
 from MDAnalysis.topology.base import TopologyReaderBase
+from MDAnalysis.core.topology import Topology
+import warnings
+
```

Comment:
```suggestion
HAS_OBABEL=False
```
Then if the import is successful, set to True, saves you having to repeat the import `try: except`

Reply:
Oh good idea! Thanks!